lib/std/mutex.zig

```diff
@@ -48,8 +48,8 @@ pub const Mutex = if (builtin.single_threaded)
             return self.tryAcquire() orelse @panic("deadlock detected");
         }
     }
-else if (builtin.os == .windows) 
-    // https://locklessinc.com/articles/keyed_events/
+else if (builtin.os == .windows)
+// https://locklessinc.com/articles/keyed_events/
```

Comment:
This doesn't look right to me. Created https://github.com/ziglang/zig/issues/3978